tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/tests/PitclipsePitSummaryViewTest.java

```diff
@@ -0,0 +1,66 @@
+package org.pitest.pitclipse.ui.tests;
+
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.swtbot.swt.finder.junit.SWTBotJunit4ClassRunner;
+import org.eclipse.swtbot.swt.finder.utils.SWTBotPreferences;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.pitest.pitclipse.ui.behaviours.pageobjects.PitSummaryView;
+import org.pitest.pitclipse.ui.view.PitView;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.endsWith;
+import static org.hamcrest.Matchers.equalTo;
+import static org.pitest.pitclipse.ui.behaviours.pageobjects.PageObjects.PAGES;
+
+@RunWith(SWTBotJunit4ClassRunner.class)
+public class PitclipsePitSummaryViewTest extends AbstractPitclipseSWTBotTest {
```

Comment:
A few more scenarios could be tested:

 - making sure the back, forward and home buttons don't break when the view is empty (no mutation has been run)
 - clicking back before clicking on a link
 - clicking forward before clicking on back

Reply:
@JKutscha I re-open because I don't see how this point is resolved. Please tell me if you think those scenarios are already addressed or explain if you believe that they don't need to be tested.

Reply:
I marked you the point in the test, which I thought resolved this comment.

If this is not sufficient, I am happy to add more or different approaches according to feedback.